includes/data-stores/class-wc-reports-orders-data-store.php

```diff
@@ -296,14 +230,7 @@ public function get_data( $query_args ) {
 				'page_no'   => (int) $query_args['page'],
 			);
 
-			if ( $this->intervals_missing( $expected_interval_count, $db_interval_count, $intervals_query['per_page'], $query_args['page'], $query_args['order'], $query_args['orderby'], count( $intervals ) ) ) {
-				$this->fill_in_missing_intervals( $db_intervals, $query_args['adj_after'], $query_args['adj_before'], $query_args['interval'], $data );
-				$this->sort_intervals( $data, $query_args['orderby'], $query_args['order'] );
-				$this->remove_extra_records( $data, $query_args['page'], $intervals_query['per_page'], $db_interval_count, $expected_interval_count, $query_args['orderby'] );
-			} else {
-				$this->update_dates( $query_args['adj_after'], $query_args['adj_before'], $query_args['interval'], $data );
-			}
-
+			$this->update_dates( $query_args['adj_after'], $query_args['adj_before'], $query_args['interval'], $data );
```

Comment:
Also not sure what `adj_after`/`adj_before` are. They don't seem to get set anywhere.

Reply:
They used to be adjusted in the final intervals <---> db intervals recalc function `update_intervals_sql_params`. However, I've just removed it, pls see also my commit.